sys/include/net/loramac.h

```diff
@@ -548,6 +563,11 @@ extern "C" {
  */
 #define LORAMAC_NETWORK_ID_LEN                  (3U)
 
+/**
+ * @brief Maximum unconfirmed uplink redundancy
+ */
+#define LORAMAC_REDUNDANCY_MAX                  (14U)
```

Comment:
The valid range in [LoRaMAC 1.0.3](https://lora-alliance.org/resource_hub/lorawan-specification-v1-0-3/) spec is 1:15 ?

Reply:
for the number of transmissions, However, here these numbers are defined as "retransmissions" (therefore 15 maps to 14)

Reply:
Yes. Maybe it makes sense to rename macro to `LORAMAC_REDUNDANCY_MAX_RETRANSMISSIONS ` or something similar ?

Reply:
as discuess offline, I can add the explanation of where the 14 comes from

Reply:
Perfect. Thanks @jia200x :)

Reply:
I removed it. I realized it's not needed because ADR already limits this

Reply:
Yes. Makes sense